src/pages/Register.tsx

```diff
@@ -1,5 +1,62 @@
+// import { useTranslation } from "react-i18next";
+import { Box, Typography, styled, TextField, Paper } from "@mui/material";
 export default function Register() {
+  const PaperStyled = styled("div")`
+    display: flex;
+    align-items: center;
+    justify-content: center;
+  `;
+
   return (
-    <div>Register</div>
-  )
+    <Box
+      component="section"
+      width="50vw"
+      sx={{ display: "flex", position: "absolute", top: "10%" }}
+    >
+      <PaperStyled>
+        <Paper elevation={3}>
+          <Typography variant="h3" color="initial" justify-content=" center">
+            Registrate
```

Comment:
Add the  useTranslation hook to include the text and add used like `const { t } = useTranslation();`

And then change the text using the react-i8next format like:
{t("register-page.title")}

and also add this new text on the translation.json file that is already added on the public/locales/es
Should be something like: 
`
"register-page": {
    "register": "Registrate",

  }
`